buildpipeline/pipelines.json

```diff
@@ -354,6 +354,39 @@
             "PB_BuildType": "Release"
           }
         },
+		{
+          "Name": "Dotnet-CoreClr-Trusted-BuildTests",
+          "Parameters": {
+            "HelixJobType": "test/functional/cli/",
+            "TargetsWindows": "true",
+            "Rid": "win-x64",
+            "TargetQueues": "Windows.10.Amd64.Core",
+            "TestContainerSuffix": "windows",
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Windows",
+            "SubType":  "Build-Tests",
+            "Type": "build/product/",
+            "PB_BuildType": "Release"
```

Comment:
I usually ask @wtgodbe for input too.  We probably want  Debug and possibly Checked legs.